web/app.js

```diff
@@ -1388,6 +1393,24 @@ const PDFViewerApplication = {
           case "error":
             console.error(detail.value);
             break;
+          case "print":
+            setTimeout(() => {
```

Comment:
Why do we need the `setTimeout` here?

Reply:
Copy/paste of:
https://github.com/mozilla/pdf.js/blob/8b652d685b4e66ea95dc036bd7aa2a200766ceb5/web/app.js#L1473